mwdb/resources/comment.py

```diff
@@ -16,6 +17,8 @@
 
 
 class CommentResource(Resource):
+    on_created = hooks.on_created_comment
+
```

Comment:
That part can be removed, especially because hook is called directly and there is no inheritance with CommentResource as a base that may reuse this method
```suggestion
```